src/views/catalog/CreateFromInstanceTypes/state/utils/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { adjectives, animals, uniqueNamesGenerator } from 'unique-names-generator';
+
+export const getRandomVMName = () =>
```

Comment:
i already think we have this function- please check

Reply:
we have - https://github.com/kubevirt-ui/kubevirt-plugin/blob/fad61ff751cd239de2b99b8a2cdf7ca84a47ed57/src/utils/resources/template/utils/helpers.ts#L45
which requires a template, I'll create a similar instance for bootable volume, sounds good?